manifests/addons/dashboards/istio-mesh-dashboard.json

```diff
@@ -430,7 +430,7 @@
       "thresholds": "",
       "timeFrom": null,
       "timeShift": null,
-      "title": "Virtual Services",
+      "title": "VirtualService",
```

Comment:
nit: i'd leave all of these both plural and with spaces for readability.

so, `Virtual Services`, `Destination Rules`, `Gateways`, `Workload Entries`, `Service Entries`. For the security bits, this gets more awkward, but I think appending `Policies` would be ok.

Reply:
Ok, I see, let me update those titles.